example3/OpenTelemetry.Metric.Sdk/Aggregators/SumCountMinMax.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+using System.Collections.Generic;
+using Microsoft.Diagnostics.Metric;
+using OpenTelemetry.Metric.Sdk;
+
+namespace OpenTelemetry.Metric.Sdk

```

Comment:
I only see aggregators in Otel SDK. Won't .NET come with a set of builtin aggregators for common things like MMSC?

Reply:
I'm interested in it but it probably doesn't matter for the sake of a prototype. The SDK (or anyone else) would always be free to implement their own aggregation logic and there is no particular benefit to using a runtime implementation other than convenience and some code sharing. Runtime aggregation wouldn't be inherently faster or better.

Reply:
My question was in context of other .net ecosystem components like Asp.Net Core/Dotnet-Counters tools which want to see aggregates, and would prefer to rely on .NET runtime to provide some built-in aggregators, so they dont need to write one (or depend on otel)

Reply:
I think we said .NET will not depend on OTel.  So, there will be some aggregators (that .NET deem important) defined only in .NET.